### ISSUE TITLE
spring-cloud-task and spring-data-jpa auto configuration cannot be used together

### ISSUE THREAD

**Issue by Ramblurr** [spring-cloud/spring-cloud-task#270]
There is a conflict when using spring-cloud-task and spring-data-jpa together with the _most simple_ auto configuration.

spring-cloud-task instantiates a transactionManager, but it is not of the type `JpaTransactionManager` that spring-data-jpa expects..

The workaround is to provide your own `JpaTransactionManager`.

Here is a minimal sample project  showing the problem including the workaround:

https://github.com/Ramblurr/spring-data-task-conflict

**Comment by Ramblurr** [comment]
Fair enough, I see the `TaskConfigurer` can be used to provide the custom transaction manager. 

But shouldn't this gotcha with spring-data be documented somewhere? I can't imagine using spring-data and spring-cloud-task is far from ordinary. Just a few words somewhere pointing out that you can't use spring-data and cloud-task out of the box together would have saved me an hour or so.

**Comment by cppwfs** [comment]
I agree.  Opened an issue to address this:  https://github.com/spring-cloud/spring-cloud-task/issues/272.
